kola/tests/update/update.go

```diff
@@ -59,6 +65,22 @@ func Serve() error {
 	return omahawrapper.Serve()
 }
 
+// assert that -update does not fail
+// when there is nothing to do
+// https://github.com/kinvolk/Flatcar/issues/356
+func idle(c cluster.TestCluster) {
```

Comment:
This test is only valid for developer builds or as long as the next release is not published. It may be confusing when running the tests for a published release.

Besides that I'm also a bit concerned about the runtime of the tests because this small test needs its own instance created and that takes multiple minutes depending on the platform.

Reply:
I think there is some support in kola for emulating an update service and to perform an update. In combination with this it makes sense to have a test where maybe in one test case an update is handed and in the other test case there is none. But I wouldn't let the behavior depend on the public update service.

Reply:
Ok then we can close this PR for the moment - I'm currently trying to think about a _better_ way to test the update behavior of Flatcar :)

Reply:
I see that we are already in the right file: `cl.update.payload` is currently not used in Jenkins, would make sense to set this up.

```
--- SKIP: cl.update.payload (19.31s)
        update.go:93: no update payload provided
```

Reply:
yeah I dug around a bit - we would need to generate an update payload with `delta_generator` and to provide it to the kola command line. I can open a dedicated issue to track this :) That would be a good start !